core/kazoo_services/src/kz_services_quantities.erl

```diff
@@ -532,7 +532,8 @@ calculate_qubicle_queue_updates(JObj, Updates) ->
     case kz_doc:type(JObj) =:= <<"qubicle_queue">> of
         'false' -> Updates;
         'true' ->
-            Key = [<<"qubicle">>, <<"queues">>],
+            Offering = kz_json:get_ne_value(<<"pvt_offering">>, JObj, <<"basic">>),
+            Key = <<Offering/binary, "_queue">>,
```

Comment:
@mark2600 the key should be:

Key = [<<"qubicle">>, <<Offering/binary, "_queue">>],